APIs/FlowCompatibilityManagementAPI.raml

```diff
@@ -89,19 +89,24 @@ mediaType: application/json
               body:
                 type: !include schemas/error.json
         put:
-          description: Replace Active Constraints with other ones.
+          description: >
+            Validates proposed Constraints against JSON Schema of request body
+            and Supported Parameter Constraints.
```

Comment:
Seems to be missing that it replace constraints .... "and upon success replace active constraints with the new ones."

Reply:
```suggestion
            and Supported Parameter Constraints and upon success
            replaces Active Constraints with the new ones.
```

Reply:
ok